docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,126 @@
+
+:::
+
+### Provisioning of data sources
+
+Data sources can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#data-sources) so that you can define Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
+
+Note that a plugin of type `datasource` must be installed before it can be provisioned.
+
+### Bundling of dashboards
+
+Data-source plugins can include dashboards by referencing dashboard JSON files (including `property` and `type=dashboard`) within the `plugin.json` file. Grafana puts a dashboard in the `General` folder when it is imported.
+
+## App plugins
+
+Development of app plugins, also known as apps or applications, allows you to create out-of-the-box solutions such as in the [Redis](https://grafana.com/grafana/plugins/redis-app/) app. You can optionally bundle data sources and panels, as well as provide custom pages, [Scenes](https://grafana.com/developers/scenes), and [UI extensions](../ui-extensions/).
+
+### Usage of app plugins
+
+Configure a Grafana app when you want to leverage or create a tailored monitoring view for a third-party service and optionally use custom pages or UI extensions. Given any installed plugin of type `app`, you can enable them once per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations) and they are persisted in Grafana’s database. 
+
+:::note
+
+To distinguish a Grafana app from an app plugin, we sometimes refer to the latter as an _app instance_, that is, a configured Grafana app with a `plugin id` type of `app`.
+
+:::
+
+### Global configuration of app plugins
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#configuration-file-location) to configure your app `[`plugin_id`](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#pluginplugin_id).
+
+:::info
+
+Not all plugins support this type of configuration. Refer to the readme of specific plugins for details.
+
+:::
+
+### Provisioning of app plugins
+
+Apps can also be provisioned using Grafana’s [provisioning features](https://grafana.com/docs/grafana/latest/administration/provisioning/#plugins) so that you can define Grafana as-code as part of a GitOps approach when Grafana starts up or [on demand](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations). 
+
+Note that the plugin must be installed before provisioning can succeed with a `plugin id` of `app` type.
+
+### Bundling of apps
+
+The app plugin type allows you to [nest other plugins inside it](../create-a-plugin/extend-a-plugin/nested-plugins); in other words, to bundle or distribute multiple plugins in the same package. 
+
+### Bundling of dashboards
```

Comment:
as above, could link to https://grafana.com/developers/plugin-tools/reference-plugin-json#includes

Reply:
ok

Reply:
as above with the clarification